api/app/Http/Controllers/ChildEntityController.php

```diff
@@ -224,7 +224,7 @@ public function deleteOne($id, $childId)
         $model = $this->findParentEntity($id);
         $childModel = $this->findOrFailChildEntity($childId, $model);
 
-        $childModel->delete();
+        $model->deleteChild($childModel, $this->relationName);
```

Comment:
@artstorm yea I understand, but I think the implementation of raising that event by having a `Model::deleteChild` method is not a clear api. I think it would be better for the trait to be able to raise events explicitly for this case eg `$model->revisionableEvent('deleteChild', $childModel)`. It is functionally the same, but it is clear why it is being done, otherwise a developer would be tempted to refactor the delete method as it doesn't appear to change anything (traits overriding class methods is a commonly difficult bug to trace).